flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/DataStreamQueryOperation.java

```diff
@@ -49,7 +50,7 @@
     private final ObjectIdentifier identifier;
     private final DataStream<E> dataStream;
     private final int[] fieldIndices;
-    private final TableSchema tableSchema;
+    private final ResolvedSchema resolvedSchema;
     // TODO remove this while TableSchema supports fieldNullables
```

Comment:
nit: I don't claim to understand this comment, but it feels like it might be out of date now :-)

Reply:
this comment still makes sense and is actually targeted at FLIP-136. We currently don't set the nullability properly in the data types of `ResolvedSchema`. I updated the comment with the new class though.